R/tm_t_pp_laboratory.R

```diff
@@ -468,8 +469,11 @@ srv_g_laboratory <- function(id,
             patient_id = patient_id()
           )
         )
-      ) %>%
-        teal.code::eval_code(as.expression(labor_calls))
+      )
+      teal.reporter::teal_card(obj) <- c(teal.reporter::teal_card(obj), "## Table")
+      obj <- obj %>% teal.code::eval_code(as.expression(labor_calls))
+      teal.reporter::teal_card(obj) <- utils::head(teal.reporter::teal_card(obj), -3) # removes table_data_html needed only for the display
```

Comment:
I don't like this here, but I don't see a good alternative as the code is core to the module and should be created in `template`.

Suggestion below to remove the linter warning

```suggestion
      # removes table_data_html needed only for the display
      teal.reporter::teal_card(obj) <- utils::head(teal.reporter::teal_card(obj), -3)

```